lib/trino-rcfile/src/main/java/io/trino/rcfile/text/TextRcFileEncoding.java

```diff
@@ -20,13 +20,14 @@
 import io.trino.spi.type.TimestampType;
 import io.trino.spi.type.Type;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class TextRcFileEncoding
         implements RcFileEncoding
 {
-    public static final byte[] DEFAULT_SEPARATORS = new byte[] {
+    private static final byte[] DEFAULT_SEPARATORS = new byte[] {
```

Comment:
Should we consider using ImmutableList instead?

Reply:
Generally yes, but the byte[] is used in other places, so that would be a bigger change, and maybe not warranted in this particular case